src/microbe_stage/AgentProjectile.cs

```diff
@@ -19,10 +28,22 @@ public void OnTimeOver()
 
     public override void _Ready()
     {
+        particles = GetNode<Particles>(ParticlesPath);
+
         AddCollisionExceptionWith(Emitter);
         Connect("body_entered", this, "OnBodyEntered");
     }
 
+    public override void _Process(float delta)
+    {
+        if (FadeTimeRemaining < 0.00001)
```

Comment:
Won't this immediately despawn this as the value starts at 0? I think the FadeTimeRemaining could be made into `float?` with null signifying that this is not currently fading.

Reply:
no, look at the whole method. I didn't use float? because I was unsure how it behaves with saving

Reply:
It should work just fine, JSON has null and number as separate types.

Reply:
I reread this and now I'm of the opinion that this is inconsistent with other places that assume that 0 denotes that no timer is currently running.